src/Stryker.Core/Stryker.Core/Baseline/Providers/AzureFileShareBaselineProvider.cs

```diff
@@ -161,7 +154,6 @@ private async Task<bool> AllocateFileLocationAsync(int byteSize, string fileUrl)
             }
             else
             {
-                _logger.LogError("Azure File Storage allocation failed with statuscode {0} and message: {1}", response.StatusCode.ToString(), ToSafeResponseMessage(await response.Content.ReadAsStringAsync()));
```

Comment:
are there no longer any errors that can occur at this point?

Reply:
Thanks for that, my mistake when deleting lines enthusiastically!